lib/http.js

```diff
@@ -272,38 +273,29 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
     routes.automatedIssue,
     // TODO: validate body
     asyncHandler(async (req, res) => {
-
-      // TODO: inspect/validate API token
-      // token will need to align with issuer.id
-
       const {
-        issuer,
-        credentials,
-      } = req.body;
-
-      // get instance for issuer
-      const {instance} = await vcIssuer.instances.get(issuer);
-      const {controller: accountId} = instance;
+        body: {credential},
+        headers: {authorization},
+      } = req;
+
+      if(!(authorization && typeof authorization === 'string' &&
+        authorization.startsWith('Bearer ') && authorization.length > 7)) {
```

Comment:
Where is the number 7 coming from? Can we declare a constant variable that tells us what this is? My assumption is that this is done to check that there is a token after `Bearer: ` in the authorization header?

Reply:
Attempted to address this in https://github.com/digitalbazaar/bedrock-vc-issuer/pull/29/commits/58f61818b1706a73032233c81546c34c2ff2b7ea

Reply:
Now using a lib and this code is eliminated.